src/Controller/FindByPid/GalleryController.php

```diff
@@ -3,27 +3,44 @@
 namespace App\Controller\FindByPid;
 
 use App\Controller\BaseController;
+use App\Ds2013\Presenters\Utilities\Paginator\PaginatorPresenter;
 use BBC\ProgrammesPagesService\Domain\Entity\Gallery;
 use BBC\ProgrammesPagesService\Domain\Entity\Image;
 use BBC\ProgrammesPagesService\Service\ImagesService;
+use BBC\ProgrammesPagesService\Service\ProgrammesAggregationService;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 class GalleryController extends BaseController
 {
     public function __invoke(
         Gallery $gallery,
         ImagesService $imagesService,
-        ?string $imagePid
+        ?string $imagePid,
+        ProgrammesAggregationService $programmesAggregationService
+
     ) {
         $this->setIstatsProgsPageType('galleries_show');
         $this->setContextAndPreloadBranding($gallery);
+        $siblingLimit = 4;
 
         $images = $imagesService->findByGroup($gallery);
         $image = $this->getFirstImage($imagePid, $images);
+        $programme= $gallery->getParent();
+        $brand = $programme->getTleo();
+        $network = $programme->getMasterBrand()->getNetwork()->getName();
+
+        $galleries = $programmesAggregationService->findDescendantGalleries($brand, $siblingLimit);
+
+
+
         return $this->renderWithChrome('find_by_pid/gallery.html.twig', [
             'gallery' => $gallery,
+            'programme' => $programme,
             'image' => $image,
             'images' => $images,
+            'network' => $network,
+            'galleries' => $galleries,
+            'brand' => $brand
```

Comment:
Add a comma or linter will fail.
```suggestion
            'brand' => $brand,
```